### ISSUE TITLE
Package bundle on Atmosphere includes outdated version of vue-meteor-tracker

### ISSUE THREAD

**Issue by jmorino** [meteor-vue/vue-meteor#216]
The bundled version installed by `meteor add akryum:vue` embeds npm package vue-meteor-tracker version 1.0.2, whereas current version is 1.2.3. This forbids to use any other version of this npm package even when installed manually via `meteor npm install --save vue-meteor-tracker`.

**Comment by Akryum** [comment]
This is strange, [it doesn't embed anything](https://github.com/meteor-vue/vue-meteor/blob/master/packages/vue/package.js). It only check if you have vue and vue-meteor-tracker in your package.json, and if not install them for you.

**Comment by jmorino** [comment]
You're right. I guess it comes from the registration process on Atmosphere.

I could work around this by cloning the repository of vue-meteor tracker inside my `packages` folder in the Meteor project's folder.
